openerp-atomfeed-service/src/test/java/org/bahmni/feed/openerp/client/OpenERPCustomerFeedIT.java

```diff
@@ -190,7 +191,7 @@ public void shouldCreateCustomerInOpenERP() throws URISyntaxException, FeedExcep
 
         when(openMRSAuthenticator.authenticate("mrsuser", "mrspwd", ObjectMapperRepository.objectMapper)).thenReturn(authenticationResponse);
         when(webClientProvider.getWebClient(any(Jobs.class))).thenReturn(webClient);
-        when(workerFactory.getWorker(Jobs.CUSTOMER_FEED, feedUrl, openERPContext, "http://mrs.auth.uri")).thenReturn(openERPCustomerServiceEventWorker);
+        when(workerFactory.getWorker(Jobs.CUSTOMER_FEED, feedUrl,endpointUrl, openERPContext, "http://mrs.auth.uri")).thenReturn(openERPCustomerServiceEventWorker);
```

Comment:
What is the difference between the `feedURL` and `endpointURL`?

Reply:
feedURL defines where the data is coming from and the other one is where is goes.

Reply:
maybe just call it then, odooURL or erpURL?

Reply:
Hey @gsluthra, Should we need to change it in everywhere or just here?

Reply:
If its easy, do it everywhere it makes sense. if change feels risky, leave it.

Reply:
We've changed it wherever we've used it. For now things look alright.